src/components/Shared/ComponentProps.tsx

```diff
@@ -0,0 +1,125 @@
+import { cx } from "class-variance-authority";
+import { PropsWithChildren, ReactNode } from "react";
+import { InlineCode } from '@/components/MDX/InlineCode'
+
+interface Props {
+  docgen: any;
+}
+
+const PropData = ({
+  title,
+  children,
+}: {
+  title: string;
+  children: ReactNode;
+}) => {
+  return (
+    <div className="flex gap-xl">
+      <p className="font-bold">{title}</p>
+      <p>{children}</p>
+    </div>
+  );
+};
+
+
+const codeStyles = cx(
+  "h-fit inline-block whitespace-nowrap rounded-small bg-transparent px-none py-none", 
+  "font-mono text-small font-regular text-info"
+)
+
+const Code = ({ children }: PropsWithChildren) => {
```

Comment:
Perhaps would be nice to separate this components into different files in the future